### ISSUE TITLE
ES6 named module import support

### ISSUE THREAD

**Issue by sukima** [jshint/jshint#2286]
According to the specs:

```text
ImportDeclaration ::= "import" ImportSpecifierSet "from" ModuleSpecifier ";"      // explicit import declaration
                   |  "import" ModuleSpecifier ";"                                // depend on the named module but don't import any bindings
```

The following syntax is correct:

```js
import 'my-module-name';
```

However JSHint complains with `esnext: true`:

> Expected '{' and instead saw 'my-module-name'.

**Comment by rwaldron** [comment]
> According to the specs:

```
ImportDeclaration ::= "import" ImportSpecifierSet "from" ModuleSpecifier ";"      // explicit import declaration
                   |  "import" ModuleSpecifier ";"                                // depend on the named module but don't import any bindings
```


That's not from the spec, but the spec does define it that way https://people.mozilla.org/~jorendorff/es6-draft.html#sec-imports

Thanks for the report.

**Comment by nicolo-ribaudo** [comment]
Fixed in 339687414e984b1f009616852d77889c0bf26ea9
